src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandlerTest.java

```diff
@@ -314,10 +314,28 @@ public void givenCaseWithEditedDwpDocsAndPhmeNotGrantedAndMultiBundleFeatureSwit
 
         PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
 
+        assertEquals(1, response.getData().getMultiBundleConfiguration().size());
+        assertEquals("bundleEnglishConfig", response.getData().getMultiBundleConfiguration().get(0).getValue());
+    }
+
+    @Test
+    public void givenCaseWithEditedDwpDocsAndPhmeUnderReviewAndMultiBundleFeatureSwitchedOn_thenReturnErrorMessageAndDoNotSendRequestToBundleService() {
```

Comment:
Can you add a scenario for when the PHME is under review please? The ticket suggests an error should be thrown in this scenario but not sure this will actually happen looking at the code, please confirm?